### PR TITLE
Search with tests

### PR DESCRIPTION
**Pull request by DuncanRose**

### CODE REVIEW

File: spec/features/recipes/user_searches_for_a_recipe_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    visit recipes_path
+
+    fill_in "search", with: "Banana"
+    click_on "Search"
+
+    expect(page).to have_content(recipe.title)
+  end
+
+  scenario 'no search results found for user search' do
```
Comment (DuncanRose) [inline spec/features/recipes/user_searches_for_a_recipe_spec.rb]: Hound is very barky today. recipe on line 8 is assigned and used on line 15 as recipe.title

File: spec/features/recipes/user_searches_for_a_recipe_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    visit recipes_path
+
+    fill_in "search", with: "Banana"
+    click_on "Search"
+
+    expect(page).to have_content(recipe.title)
+  end
+
+  scenario "no search results found for user search" do
+    recipe = FactoryGirl.create(:recipe)
```
Comment (DuncanRose) [inline spec/features/recipes/user_searches_for_a_recipe_spec.rb]: Hound is barking up the wrong tree. recipe is assigned and used on line 15.....

File: spec/features/recipes/user_searches_for_a_recipe_spec.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (flourishcoaching) [inline spec/features/recipes/user_searches_for_a_recipe_spec.rb]: I don't see where you use recipe on the scenario on line 17, but that's not a high impact edit. No worries.

File: spec/features/recipes/user_searches_for_a_recipe_spec.rb (hunk 2 above)
Comment (omidbachari) [inline spec/features/recipes/user_searches_for_a_recipe_spec.rb]: I don't see where you use recipe on the scenario on line 17, but that's not a high impact edit. No worries.

**Pull request merged**